components/802.15.4_RF/atmel-rf-driver/source/NanostackRfPhyAtmel.cpp

```diff
@@ -112,8 +112,8 @@ static uint8_t mac_tx_handle = 0;
 static uint8_t xah_ctrl_1;
 
 /* Channel configurations for 2.4 and sub-GHz */
-static const phy_rf_channel_configuration_s phy_24ghz = {2405000000U, 5000000U, 250000U, 16U, M_OQPSK};
-static const phy_rf_channel_configuration_s phy_subghz = {868300000U, 2000000U, 250000U, 11U, M_OQPSK};
+static const phy_rf_channel_configuration_s phy_24ghz = {.channel_0_center_frequency = 2405000000U, .channel_spacing = 5000000U, .datarate = 250000U, .number_of_channels = 16U, .modulation = M_OQPSK};
+static const phy_rf_channel_configuration_s phy_subghz = {.channel_0_center_frequency = 868300000U, .channel_spacing = 2000000U, .datarate = 250000U, .number_of_channels = 11U, .modulation = M_OQPSK};
```

Comment:
This is C99, but should not be legal in C++
https://stackoverflow.com/questions/18731707/why-does-c11-not-support-designated-initializer-lists-as-c99